test/package/tsc/package.json

```diff
@@ -16,5 +16,8 @@
   "devDependencies": {
     "rimraf": "^2.6.2",
     "typescript": "^4.2.2"
+  },
+  "resolutions": {
```

Comment:
This may not be necessary with authJS 6.7.3.  The miocrotime dependency was updated in broadcast-channel which caused a minimum node version bump.

https://github.com/okta/okta-auth-js/pull/1262/files

It is OK to leave this here for now, but we should clean up when we upgrade authJS